requirements-test.txt

```diff
@@ -0,0 +1,8 @@
+flake8>=2.1.0,<2.2.0
```

Comment:
I'm generally -1 on this pattern. setup.py is a subset of requirements, and its already possible to do everything thats needed from there. The only defense I can imagine here is "because Docker", and if that's the case I'm willing to consider it, but otherwise this just seems to be change because change.